app/navigation/Navigation.scala

```diff
@@ -65,7 +65,8 @@ class Navigation @Inject()(dateTimeHelper: DateTimeHelper,
     CheckYourAnswersPage -> ((request, jsEnabled) => reverseRouteForCYAPage(request, NormalMode, jsEnabled)),
     PenaltySelectionPage -> ((_, _) => routes.AppealStartController.onPageLoad()),
     AppealSinglePenaltyPage -> ((_, _) => routes.PenaltySelectionController.onPageLoadForPenaltySelection(NormalMode)),
-    AppealCoverBothPenaltiesPage -> ((_, _) => routes.PenaltySelectionController.onPageLoadForPenaltySelection(NormalMode))
+    AppealCoverBothPenaltiesPage -> ((_, _) => routes.PenaltySelectionController.onPageLoadForPenaltySelection(NormalMode)),
+    AppealByLetterKickOutPage -> ((_, _) => routes.YouCannotAppealController.onPageLoadAppealByLetter())
```

Comment:
Not sure this is right, this would mean clicking the back would technically just reload the page. It should go back to the previous page.

If you load the page up and click the back link on the page you'll see what I mean